ui/src/app/records/search/brief-view/documents-brief-view.component.ts

```diff
@@ -61,9 +62,25 @@ export class DocumentsBriefViewComponent implements BriefView {
 
   constructor(
     private recordsService: RecordsService,
-    private alertsService: AlertsService
+    private alertsService: AlertsService,
+    private translate: TranslateService
     ) {}
 
+  publisherNames() {
+    const name_index = 'name_' + this.translate.currentLang;
+    const publishers = [];
+    for (const publisher of this.record.metadata.publishers) {
+      let name_lng = publisher[name_index];
+      if (!name_lng) {
+        name_lng = publisher['name'];
+      }
+      if (name_lng) {
```

Comment:
please use `else`

Reply:
Can't replaced with else, but we can delete the second if.